src/menhir_parser.mly

```diff
@@ -282,4 +282,8 @@ term:
   | x=ID COLON a=aterm FARROW t=term {
       make_pos $loc (P_Abst([(make_pos $loc(x) x, Some(a))], t))
     }
+  | L_PAR x=ID COLON t=aterm DEFEQ b=aterm R_PAR FARROW u=term {
```

Comment:
Why do you change the grammar? It doesn't seem related to termination checking.

Reply:
Indeed, I branched on the wrong commit, I will change this right now.